lib/services/OrderService.js

```diff
@@ -7,9 +7,25 @@ module.exports = class OrderService {
       process.env.ORDER_HANDLER_NUMBER,
       `New Order received for ${value.quantity}`
     );
-
     const order = await Order.insert(value);
+    return order;
+  }
+
+  static async updateOrder(value) {
+    await sendSms(
+      process.env.ORDER_HANDLER_NUMBER,
+      `Order ${value.id}, updated quantity: ${value.quantity}`
+    );
+    const order = await Order.update(value);
+    return order;
+  }
 
+  static async deleteOrder(value) {
+    await sendSms(
+      process.env.ORDER_HANDLER_NUMBER,
+      `Cancelled order ${value.id}`
+    );
+    const order = await Order.delete(value);
```

Comment:
Same thing here, this method doesn't exist on your `Order` model, but `deleteById` will do what you need.